x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupRequestTranslator.java

```diff
@@ -220,7 +220,7 @@
                 rolledDateHisto.interval(source.interval());
             }
 
-            String timezone = source.timeZone() == null ? DateTimeZone.UTC.toString() : source.timeZone().toString();
+            String timezone = source.timeZone() == null ? ZoneOffset.UTC.toString() : source.timeZone().toString();
```

Comment:
does this get serialized over the wire somewhere? If so we may need to convert it using `DateUtils`?

Reply:
this is read in `ValuesSourceAggregationBuilder` so I guess it is already taken care of?
with
```
 if (in.getVersion().before(Version.V_7_0_0)) {
            timeZone = DateUtils.dateTimeZoneToZoneId(in.readOptionalTimeZone());
        } else {
            timeZone = in.readOptionalZoneId();
        }
```

Reply:
++